package.json

```diff
@@ -28,10 +28,10 @@
     "@datapunt/matomo-tracker-react": "^0.3.1",
     "@date-io/date-fns": "^1.3.13",
     "@inrupt/prism-react-components": "^0.13.7",
-    "@inrupt/solid-client-access-grants": "^0.3.3-fix2258-verification-endpoint-discovery-1412767200-263-1635864188.0",
-    "@inrupt/solid-client": "^1.15.0",
-    "@inrupt/solid-client-authn-browser": "^1.8.2",
-    "@inrupt/solid-ui-react": "^2.3.1",
+    "@inrupt/solid-client": "^1.16.1",
+    "@inrupt/solid-client-access-grants": "^0.3.3-release-1398419875-258-1635500877.0",
```

Comment:
We'll soon cut a release of that library, probably v0.4.0, but this can be updated later no need to wait on it.